models/UserModel.js

```diff
@@ -1,14 +1,28 @@
 var mongoose = require("mongoose");
+const { constants } = require("../helpers/constants");
 
-var UserSchema = new mongoose.Schema({
-	name: { type: String, required: true },
-	email: { type: String, required: true },
-	password: { type: String, required: true },
-	isConfirmed: { type: Boolean, required: true, default: 0 },
-	confirmOTP: { type: String, required: false },
-	otpTries: { type: Number, required: false, default: 0 },
-	status: { type: Boolean, required: true, default: 1 },
-	userType: { type: String, required: true },
-}, { timestamps: true });
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
 
-module.exports = mongoose.model("User", UserSchema);
\ No newline at end of file
+var UserSchema = new Schema(
+  {
+    name: { type: String, required: true },
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
		name: { type: String, required: true },
```